lib/MusicBrainz/Server/Entity/URL/Allmusic.pm

```diff
@@ -5,6 +5,10 @@ use Moose;
 extends 'MusicBrainz::Server::Entity::URL';
 with 'MusicBrainz::Server::Entity::URL::Sidebar';
 
+override href_url => sub {
+    shift->url->as_string =~ s{^http:}{https:}r;
```

Comment:
I guess it can't hurt, but I'm mainly wondering why it can't wait for the script to finish at this point. :)

Reply:
Not to keep editors waiting for about one more month after server update has been announced, and not to return links with mixed scheme during that time. :watch: I can burden myself with reverting this commit in one or two server updates. :weight_lifting_man:

Reply:
In general it seems fine to me, although this temporary change seems like a lot of work for little use. But since you've already done it, I guess it doesn't hurt at the moment :)

Reply:
Just merged other PR into master rebased this one on master.